container/id/id.go

```diff
@@ -120,3 +121,9 @@ func (id ID) IsZero() bool {
 	}
 	return true
 }
+
+// Compare performs three-way comparison of [ID] and returns the result. It
```

Comment:
have been thinking for ~1 min about whether it is an exhaustive description. Likely it is ok, but when i use GO's std libs i always compare with `-1`, `+1`, but there it should not be considered as a correct comparison since it is not mentioned